src/PhpWord/Style.php

```diff
@@ -95,7 +95,7 @@ public static function addNumberingStyle($styleName, $styleValues)
      */
     public static function addTitleStyle($depth, $fontStyle, $paragraphStyle = null)
     {
-        if ($depth === null) {
+        if ($depth === null || $depth === '' || $depth === 0) {
```

Comment:
Could we just use `empty($depth)` here? it would catch all these cases.

Reply:
indeed, that would definitely be cleaner